emoji/src/main/java/com/vanniktech/emoji/EmojiSkinTonePopup.java

```diff
@@ -0,0 +1,91 @@
+package com.vanniktech.emoji;
+
+import android.app.Activity;
+import android.graphics.Bitmap;
+import android.graphics.drawable.BitmapDrawable;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.view.Gravity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.WindowManager;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.PopupWindow;
+
+import com.vanniktech.emoji.emoji.Emoji;
+import com.vanniktech.emoji.listeners.OnEmojiClickedListener;
+
+import java.util.List;
+
+class EmojiSkinTonePopup {
+  private static final int MARGIN = 4;
+
+  private PopupWindow popupWindow;
+
+  @Nullable private final OnEmojiClickedListener listener;
+
+  EmojiSkinTonePopup(@Nullable final OnEmojiClickedListener listener) {
+    this.listener = listener;
+  }
+
+  void show(@NonNull final View clickedImage, @NonNull final Emoji emoji) {
+    dismiss();
+
+    final View content = View.inflate(clickedImage.getContext(), R.layout.emoji_skin_popup, null);
+    final LinearLayout imageContainer = (LinearLayout) content.findViewById(R.id.container);
+    popupWindow = new PopupWindow(content,
+            WindowManager.LayoutParams.WRAP_CONTENT,
+            WindowManager.LayoutParams.WRAP_CONTENT
+    );
+
+    final List<Emoji> skinTonedEmojis = EmojiManager.getInstance().findSkinTonedEmojis(emoji);
+    skinTonedEmojis.add(0, emoji);
+
+    for (final Emoji tonedEmoji : skinTonedEmojis) {
+      final ImageView emojiImage = (ImageView) LayoutInflater.from(clickedImage.getContext())
+              .inflate(R.layout.emoji_item, imageContainer, false);
+      final ViewGroup.MarginLayoutParams layoutParams = (ViewGroup.MarginLayoutParams) emojiImage.getLayoutParams();
+      final int margin = (int) Utils.dpToPx(clickedImage.getContext(), MARGIN);
```

Comment:
`dpToPx` returns `float`, but we need `int` here.
The code is from [this](http://stackoverflow.com/a/12147550/4279995) StackOverflow answer.